verticaltabs.js

```diff
@@ -428,6 +444,11 @@ VerticalTabs.prototype = {
     let previous_close_message = close_next_tabs_message.getAttribute('label');
     close_next_tabs_message.setAttribute('label', 'Close Tabs Below');
 
+    //remove option to hide new-tab-button
+    let parentMenu = document.getElementById('toolbar-context-menu');
+    let moveToPanelMenuItem = parentMenu.removeChild(document.querySelector('.customize-context-moveToPanel'));
+    let removeFromToolbarMenuItem = parentMenu.removeChild(document.querySelector('.customize-context-removeFromToolbar'));
```

Comment:
This seems to remove the "Move to Menu" and "Remove from Toolbar" options from _all_ the buttons in the urlbar, not just the newtab button…  Can we instead override `onViewToolbarsPopupShowing` from https://dxr.mozilla.org/mozilla-central/source/browser/base/content/browser.xul#275 to only disable them for the newtab button?  (And I _think_ disabling might be better than removing…  I'm not sure, though.)